pydash/utilities.py

```diff
@@ -393,6 +394,36 @@ def noop(*args, **kargs):  # pylint: disable=unused-argument
     pass
 
 
+def nth_arg(pos=0):
+    """Creates a function that gets the argument at index n. If n is negative,
+    the nth argument from the end is returned.
+
+    Args:
+        pos (int): The index of the argument to return.
+
+    Returns:
+        function: Returns the new pass-thru function.
+
+    Example:
+
+        >>> func = nth_arg(1)
+        >>> func(11, 22, 33, 44)
+        22
+        >>> func = nth_arg(-1)
+        >>> func(11, 22, 33, 44)
+        44
+
+    .. versionadded:: TODO
+    """
+    def _nth_arg(*args):
+        if args and (len(args) > pos >= -len(args)):
```

Comment:
You can just do a try/except and catch an IndexError. Or you can use pyd.get() which handles this behavior.